systemtest/src/test/java/io/strimzi/systemtest/UserST.java

```diff
@@ -114,6 +117,26 @@ void testUpdateUser() {
         StUtils.waitForKafkaUserDeletion(kafkaUser);
     }
 
+    @Tag(PERFORMANCE)
+    @Test
+    void testBigAmountOfUsers(){
+        int numberOfUsers = 100;
+
+        for(int i = 0; i < numberOfUsers; i++){
+            String userName = "alisa" + i;
+            LOGGER.info("Creating user with name {}", userName);
+            testMethodResources().tlsUser(CLUSTER_NAME, userName).done();
+            StUtils.waitForSecretReady(userName);
+            LOGGER.info("Checking status of deployed Kafka User {}", userName);
+            Condition kafkaCondition = testMethodResources().kafkaUser().inNamespace(NAMESPACE).withName(userName).get()
+                    .getStatus().getConditions().get(0);
+            LOGGER.info("Kafka User Status: {}", kafkaCondition.getStatus());
+            LOGGER.info("Kafka User Type: {}", kafkaCondition.getType());
```

Comment:
Actually you are not logging the user type but the user status condition type.

Reply:
fixed